crates/sui-core/src/consensus_validator.rs

```diff
@@ -91,13 +85,12 @@ impl SuiTxValidator {
                 | ConsensusTransactionKind::RandomnessStateUpdate(_, _) => {}
 
                 ConsensusTransactionKind::UserTransaction(_tx) => {
-                    if !self.epoch_store.protocol_config().mysticeti_fastpath() {
+                    if !epoch_store.protocol_config().mysticeti_fastpath() {
                         return Err(SuiError::UnexpectedMessage(
                             "ConsensusTransactionKind::UserTransaction is unsupported".to_string(),
                         )
                         .into());
                     }
-                    // TODO(fastpath): implement verification for uncertified user transactions.
```

Comment:
why is this todo no longer required? Is it because we verify when we vote?

Reply:
Yeah I was thinking the verifications in `vote_transactions()` make this TODO obsolete. But it may be better to still keep track of the TODO to move those checks here.